src/components/workers-page/shifts-tab/shifts-tab.component.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function ShiftTab(): JSX.Element {
+  return <h2>Zmiany</h2>;
```

Comment:
Zastanawiam się nad tym, czy nie zrobić teraz  jakiś `ResoruceProvider` i do niego dodawać napisy?
Bo jeżeli potem będziemy chcieli wprowadzić potem nowy język (wierzę w to, że dojdziemy do takiej skali :D ), to z `ResoruceProviderem` będzie to robić zdecydowanie łatwiej.

Reply:
Trzeba ogarnąć jak się robi internacjonalizacje w React'cie, ale jestem na tak :D